src/catalog/catalog_entry/table_catalog_entry.cpp

```diff
@@ -176,6 +176,24 @@ string TableCatalogEntry::ColumnsToSQL(const ColumnList &columns, const vector<u
 	return ss.str();
 }
 
+string TableCatalogEntry::ColumnNamesToSQL(const ColumnList &columns) {
+	if (columns.empty()) {
+		return "";
+	}
+
+	std::stringstream ss;
+	ss << "(";
+
+	for (auto &column : columns.Logical()) {
+		if (column.Oid() > 0) {
+			ss << ", ";
+		}
+		ss << column.Name();
```

Comment:
This is likely missing `KeywordHelper::WriteOptionallyQuoted` - can we add a test where we use column names that require quoting (e.g. `"Column Name"`)?